### PR TITLE
Enable InstantOn for appAuthorization

### PR DESCRIPTION
**Pull request by anjumfatima90**
for #28205

### CODE REVIEW

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Review by SmithaSubbarao — COMMENTED** [review]
Based on the changes in this PR and my current understanding, it looks like checkpoint is being enabled for appAuthorization features and also updating the latest version of appAuthorization across all tests (v2.1) and related dependencies. 

In addition to this, there is EEXfeatureReplacement happening across various versions of Java across the tests as mentioned here in this line: https://github.com/OpenLiberty/open-liberty/pull/29038/files#diff-cbae871be5f4510ee3f62b8e916f370299a9d0e594f75afbd535a231f6d24036R176 

One question I have is, regarding the difference in actions between EE8FeatureReplacementAction and JakartaEE9Action and  -> I understand that we are running the EE9 and EE10 tests in LITE mode for Java 8 and versions greater than to equal to Java 11 but just wanted to know if we are replacing the EE8 features across all versions. Thank you!

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer.security.jacc_fat.2.1/publish/servers/com.ibm.ws.ejbcontainer.security.jacc_fat.ejbjar/server.xml [hunk 1]
```diff
@@ -20,6 +20,9 @@
 	</featureManager>
 	
 	<jaccTestProvider roleMappingFile="${server.config.dir}/resources/security/roleMapping.props" />
+	
+	<!--startEJBsAtAppStart is false by default. But in the case of checkpoint/restore, the default value is true. hence, setting false explicitly -->
+	<ejbContainer startEJBsAtAppStart="false"/>
```
Comment (tjwatson) [inline dev/com.ibm.ws.ejbcontainer.security.jacc_fat.2.1/publish/servers/com.ibm.ws.ejbcontainer.security.jacc_fat.ejbjar/server.xml:25]: Is there an issue to track a fix for this such that setting this to `true` allows it to work for checkpoint.

**Review by tjwatson — APPROVED** [review]

File: dev/com.ibm.ws.ejbcontainer.security.jacc_fat.2.1/publish/servers/com.ibm.ws.ejbcontainer.security.jacc_fat.ejbjar/server.xml (hunk 1 above)
Comment (anjumfatima90) [reply dev/com.ibm.ws.ejbcontainer.security.jacc_fat.2.1/publish/servers/com.ibm.ws.ejbcontainer.security.jacc_fat.ejbjar/server.xml:25]: @tkburroughs, last time we talked, you suggested there could be a bug here that is unrleated to checkpoint.

**Review by SmithaSubbarao — APPROVED** [review]
/lgtm

**Comment by anjumfatima90** [comment]
#libby

**Pull request merged**